packages/frontend/src/stores/catalog.ts

```diff
@@ -0,0 +1,24 @@
+import type { Readable } from 'svelte/store';
+import { readable } from 'svelte/store';
+import { MSG_NEW_CATALOG_STATE } from '@shared/Messages';
+import { rpcBrowser, studioClient } from '/@/utils/client';
+import type { Catalog } from '@shared/src/models/ICatalog';
```

Comment:
it feels odd to me to have ICatalog file with a Catalog and not a ICatalog
same for ITask.ts having Task object

Might be easier to have as convention the ICatalog for interface name as well so we know if we imports somehow an interface or an object

or you name the file Catalog not ICatalog else I don't see the point of prefixing the filename

Reply:
@axel7083 what do you think? As you started with this convention, would you prefer to use `I` everywhere (file names and interface names) or remove them from everywhere?

Reply:
I would not mind either ones.

I would have a personal preference for having file with `I` prefix and interface name as it improve readability and expectation

Reply:
OK, I'll rename the interfaces to add an `I` prefix, so the file names and interafce names are the same. I'll do this in an upcoming PR